.github/workflows/scala-steward.yml

```diff
@@ -0,0 +1,16 @@
+name: Scala Steward
```

Comment:
Other repos in this organisation are using the public Scala-steward, by adding the repo here: https://github.com/search?q=repo%3AVirtusLab%2Fscala-steward-repos%20sbt%2Fsbt-&type=code

Reply:
IMO, it's better to use the Github Actions. It's local to the repo, you have more control to when it's launched, you can launch it manually if needed and the PRs don't come from a fork so they're easier to update if necessary

Reply:
Makes sense